hydra-node/src/Hydra/API/ServerOutput.hs

```diff
@@ -112,13 +112,15 @@ instance IsChainState tx => ToJSON (Greetings tx) where
     genericToJSON
       defaultOptions
         { omitNothingFields = True
+        , tagSingleConstructors = True
```

Comment:
This looks unrelated to the thing you are trying to do. I vote to remove it if there is no real need for it.

Reply:
it is unrelated but required to have a cleaner e2e test.

please note all our outputs have tags, this was the only missing one.
for consistency i would recommend keeping it.